application/recognition/image_recognition.py

```diff
@@ -0,0 +1,56 @@
+"""Image Recognition"""
+from eventlet import spawn_n
+from requests import post
+from application.schemas.artifact_schema import ArtifactSchema
+from flask import current_app
+
+
+class ImageRecognizer():
+    @staticmethod
+    def auto_add_tags(artifact):
+        """Called by classes """
+        spawn_n(ImageRecognizer._work_asynchronously, artifact)
+
+    @staticmethod
+    def _call_google_api(url):
+        """Does API call to could vision api"""
+        api_url = current_app.config.get('CLOUD_VISION_API_URL')
+        api_key = current_app.config.get('CLOUD_VISION_API_KEY')
+        r = post(url=api_url,
+                 data={
+                     "requests": [
+                         {
+                             "features": [
+                                 {"type": "LABEL_DETECTION"},
+                                 {"type": "TEXT_DETECTION"}
+                             ],
+                             "image": {
+                                 "source": {"imageUri": url}
+                             }
+                         }
+                     ]
+                 },
+                 params={'key': api_key})
+        return r.json()
+
+    @staticmethod
```

Comment:
Not a huge fan of having a class with only static methods. I'd prefer most of these to simply be class methods so if you want to use class variables or later on intialize objects you easily can. The difference is miniscule but for example a static private function doesn't make much sense. Since the static function doesn't even belong to a class how is it supposed to be private? A private function should not be called from outside the class but the static method doesn't even belong to a class so every call is from outside...  
Just feels not right to me at least.